src/index.tsx

```diff
@@ -32,6 +32,7 @@ import { ThemeProvider } from './theme/ThemeProvider'
 export { ProjectRole } from './lib/roles'
 export type { state } from './store'
 export * from './store/ParentObserver'
+export { getUserRole } from './lib/roles'
```

Comment:
Maybe I'm missing something but I don't see it being used.

Reply:
removed